Drv/LinuxSpiDriver/LinuxSpiDriverComponentImpl.hpp

```diff
@@ -59,7 +77,8 @@ namespace Drv {
             //! Open device
             bool open(NATIVE_INT_TYPE device,
                       NATIVE_INT_TYPE select,
-                      SpiFrequency clock);
+                      SpiFrequency clock,
+                      SpiMode spiMode);
```

Comment:
Should we add a default parameter here such that old code still works?

`spiMode = SPI_MODE_0`